website/content/docs/agent/caching/index.mdx

```diff
@@ -134,7 +134,8 @@ belonging to the cached response, the `request_path` that resulted in the
 cached response, the `lease` that is attached to the cached response, the
 `namespace` to which the cached response belongs to, and a few more. This API
 exposes some factors through which associated cache entries are fetched and
-evicted.
+evicted. For listeners without caching enabled, this API will still be available,
+but will do nothing (there is no cache to clear) and will return a `200` response.
```

Comment:
Q: When caching is enabled, does the call to the API endpoint return any content in the response body? 

If **yes**, I think we should adjust the 'disabled caching' response code to `204 (No Content)`.
If **no**, I'm not sure if that's a bit weird semantically but hey ho, I guess it makes no difference if we 'keep it the same' here.

Reply:
It doesn't return content in either case (caching or no), but returns a `200`. I'd broadly agree that it 'should' be a `204`, but my goal was to keep the return values the same between the two. If we changed the cache endpoint to `204`, we should update this one too.